### ISSUE TITLE
paragraph math equations without numbers, including align environment

### ISSUE THREAD

**Issue by iaacornus** [iaacornus/simtex#13]

**Comment by iaacornus** [comment]
in the recent commit, `--` was introduced to denote \nonumber for mathmode. So \nonumber can be toggled, e.g.

```
$$
P(t) = \frac{K}{1 + \bigg(\frac{K - P_{0}}{P_{0}}\bigg)e^{-kt}} 
P(t) \cdot 1 + \bigg(\frac{K - P_{0}}{P_{0}}\bigg)e^{-kt} = K --
1 + \bigg(\frac{K - P_{0}}{P_{0}}\bigg)e^{-kt} = \frac{K}{P(t)} --
\bigg(\frac{K - P_{0}}{P_{0}}\bigg)e^{-kt} = \frac{K}{P(t)} - 1 \\--
e^{-kt} = \frac{\frac{K}{P(t)} - 1}{\bigg(\frac{K - P_{0}}{P_{0}}\bigg)} \longrightarrow \frac{\frac{K}{P(t)} - 1}{A} --\\
= \frac{\frac{K - P(t)}{P(t)}}{A}
e^{-kt} = \frac{K - P(t)}{P(t) \cdot A}
ln(e^{-kt}) = ln\bigg(\frac{K - P(t)}{P(t) \cdot A}\bigg)
-kt = ln\bigg(\frac{K - P(t)}{P(t) \cdot A}\bigg)
$$
```

Where this should be converted to:

```
	\begin{align}
		P(t) &= \frac{K}{1 + \bigg(\frac{K - P_{0}}{P_{0}}\bigg)e^{-kt}} \\
		P(t) \cdot 1 + \bigg(\frac{K - P_{0}}{P_{0}}\bigg)e^{-kt} &= K \nonumber \\
		1 + \bigg(\frac{K - P_{0}}{P_{0}}\bigg)e^{-kt} &= \frac{K}{P(t)} \nonumber \\
		\bigg(\frac{K - P_{0}}{P_{0}}\bigg)e^{-kt} &= \frac{K}{P(t)} - 1 \nonumber \\
		e^{-kt} &= \frac{\frac{K}{P(t)} - 1}{\bigg(\frac{K - P_{0}}{P_{0}}\bigg)} \longrightarrow \frac{\frac{K}{P(t)} - 1}{A} \nonumber \\
		&= \frac{\frac{K - P(t)}{P(t)}}{A} \\
		e^{-kt} &= \frac{K - P(t)}{P(t) \cdot A} \\
		ln(e^{-kt}) &= ln\bigg(\frac{K - P(t)}{P(t) \cdot A}\bigg) \\
		-kt &= ln\bigg(\frac{K - P(t)}{P(t) \cdot A}\bigg) 
	\end{align}
```

**Comment by iaacornus** [comment]
`\text{}` should also be automatically be numberless, and there should be a feature to automatically make the whole align env numberless

**Comment by iaacornus** [comment]
in cases of align, `$$--` (eefbd855a143d112c409b60d5253ab16ad5781c5) can be used to denote numberless align environment which was implemented in f6a82609585891505b72f72766e89d374b73a907
